optics-core/src/Optics/Internal/Profunctor.hs

```diff
@@ -256,66 +256,77 @@ class (Choice p, Strong p) => Traversing p where
   wander
     :: (forall f. Applicative f => (a -> f b) -> s -> f t)
     -> p i a b -> p i s t
-
   iwander
     :: (forall f. Applicative f => (i -> a -> f b) -> s -> f t)
     -> p j a b -> p (i -> j) s t
 
+  conjoined
```

Comment:
Note: this method can be in `Profunctor`.

Reply:
I'll add a comment. We can move it along with ixcontramap, as only then it'll actually be useful there.